Sources/Apodini/Semantic Model Builder/OpenAPI/Document/OpenAPIPathsObjectBuilder.swift

```diff
@@ -52,6 +52,9 @@ struct OpenAPIPathsObjectBuilder {
 private extension OpenAPIPathsObjectBuilder {
     /// https://swagger.io/specification/#operation-object
     mutating func buildPathItemOperationObject<H: Handler>(from endpoint: Endpoint<H>) -> OpenAPI.Operation {
+        // Set operationId to concatenated string of operation and `Handler`s type name
+        let operationId = "\(endpoint.description)"
+        
```

Comment:
just one small point: the naming of `operationId` seems a bit off to me when assigning a description to it.

Reply:
464206c